diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index cf9bd1a..d58cc64 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -145,6 +145,7 @@ buildscript {
         classpath "com.android.tools.build:gradle:8.2.1"
         classpath "de.undercouch:gradle-download-task:5.6.0"
         classpath "com.diffplug.spotless:spotless-plugin-gradle:6.25.0"
+        ext.kotlin_version = "1.9.25"
     }
 }
 
@@ -155,9 +156,10 @@ if (project == rootProject) {
 apply plugin: "com.android.library"
 apply plugin: "maven-publish"
 apply plugin: "de.undercouch.download"
+apply plugin: "kotlin-android"
 
 android {
-    compileSdkVersion safeExtGet("compileSdkVersion", 34)
+    compileSdkVersion safeExtGet("compileSdkVersion", 36)
 
     namespace "com.swmansion.reanimated"
 
@@ -181,8 +183,8 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet("minSdkVersion", 23)
-        targetSdkVersion safeExtGet("targetSdkVersion", 34)
+        minSdkVersion safeExtGet("minSdkVersion", 24)
+        targetSdkVersion safeExtGet("targetSdkVersion", 36)
         versionCode 1
         versionName REANIMATED_VERSION
 
